roles/wait_for_active_he_vm/files/wait_for_active_he_vm

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env python
+
+import time
+from ovirtsdk.api import API
+
+
+def main():
+    args = parse_args()
+    engine_api = get_api(
+        url=args.url,
+        user=args.username,
+        password=args.password,
+        insecure=True
+    )
+    wait_for_he_vm_up_state(engine_api)
+
+
+def parse_args():
+    import argparse
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--url', dest='url', required=True,
+                        help='the url for the Engine api')
+    parser.add_argument('--username', dest='username', required=True,
+                        help='the username for the Engine api')
+    parser.add_argument('--password', dest='password', required=True,
+                        help='the password for the Engine api')
+    args = parser.parse_args()
+    return args
+
+
+def get_api(url=None, user=None, password=None, insecure=False, timeout=30):
+    start = time.time()
+    while time.time() - start < timeout:
+        try:
+            return API(
+                url=url,
+                username=user,
+                password=password,
+                insecure=insecure
```

Comment:
hmmm... shouldn't there be an option to do it securely, or in this case it doesn't matter?